src/components/Admin/index.jsx

```diff
@@ -365,16 +307,28 @@ class Admin extends React.Component {
                 <div className="row">
                   <div className="col">
                     {!error && !loading && !this.hasEmptyData() && (
-                      <div className="row">
-                        <div className="col-12 col-md-12 col-lg-12 col-xl-4 pt-1 pb-3">
-                          {lastUpdatedDate &&
-                            <React.Fragment>
-                              Showing data as of {formatTimestamp({ timestamp: lastUpdatedDate })}
-                            </React.Fragment>
-                          }
+                      <React.Fragment>
+                        <div className="row pb-3">
+                          <div className="col-12 col-md-6  col-xl-4 pt-1 pb-3">
+                            {lastUpdatedDate &&
+                              <React.Fragment>
+                                Showing data as of {formatTimestamp({ timestamp: lastUpdatedDate })}
```

Comment:
I can't see this text on the ui.. how would I test this piece?

Reply:
Honestly, I don't know what makes `lastUpdatedDate` show up. (It's not my original code, just moved tot this location.)
For UI testing I just changed the `lastUpdatedDate` before the and to `true`.